pkg/proxy/iptables/proxier.go

```diff
@@ -1620,6 +1620,12 @@ func (proxier *Proxier) syncProxyRules() {
 
 	for name, lastChangeTriggerTimes := range endpointUpdateResult.LastChangeTriggerTimes {
 		for _, lastChangeTriggerTime := range lastChangeTriggerTimes {
+			// skip the endpoints with zero value, those are the ones without
+			// the EndpointsLastChangeTriggerTime annotation, so it is impossible
+			// to estimate the latency.
+			if lastChangeTriggerTime.IsZero() {
+				continue
```

Comment:
add some warn log?
And Can you share your logs or metrics data of kube-proxy.

Reply:
I need to revisit it , this is not the problem ... :/